src/pieces/__initi__.py

```diff
@@ -0,0 +1,26 @@
+# Importing the WebSocketManager class
```

Comment:
I've exposed nearly every funciton in the library here, prob want to be more selective